### ISSUE TITLE
Chrome or LXDE required?

### ISSUE THREAD

**Issue by sbrooke** [grafana/grafana-kiosk#111]
Per this part of the install page:

_This application can run on most operating systems, but for linux some additional binaries are suggested for full support.
Suggesting Packages:
unclutter (for hiding mouse/cursor) rng-tools (for entropy issues)_

There's no mention of a desktop environment or Chrome(ium) being required. When I try to run it I get an error message about not being able to locate the google-chrome executable.

It's fine if it requires a desktop or X environment of some kind. The documentation doesn't reflect that.

**Comment by briangann** [comment]
Yes, the wayland modifications will be included in the near future. I have an Orange pi and Libre pi that are running it, and have been useful alternatives for kiosk devices.

**Comment by llamafilm** [comment]
Did you find a good way to hide the mouse cursor in Wayland?
